core/src/main/java/org/elasticsearch/common/transport/InetSocketTransportAddress.java

```diff
@@ -34,41 +33,22 @@
  */
 public final class InetSocketTransportAddress implements TransportAddress {
 
-    // TODO: do we really need this option, why do resolving? - remove this as a follow-up
-    private static boolean resolveAddress = false;
-
-    public static void setResolveAddress(boolean resolveAddress) {
-        InetSocketTransportAddress.resolveAddress = resolveAddress;
-    }
-
-    public static boolean getResolveAddress() {
-        return resolveAddress;
-    }
-
-    public static final InetSocketTransportAddress PROTO = new InetSocketTransportAddress();
+    public static final InetSocketTransportAddress PROTO = new InetSocketTransportAddress(new InetSocketAddress("127.0.0.1", 0));
```

Comment:
What uses this? And why is forbidden APIs not angry about passing in String,int here... i feel like i banned that method. I dont like it as its wired to 127.0.0.1 in non-test code.